ooniprobe.xcodeproj/project.pbxproj

```diff
@@ -1366,7 +1372,7 @@
 				STRIP_INSTALLED_PRODUCT = NO;
 				TARGETED_DEVICE_FAMILY = "1,2";
 				USER_HEADER_SEARCH_PATHS = "";
-				VALID_ARCHS = "armv64 armv7s i386 x86_64";
+				VALID_ARCHS = "arm64 armv7s i386 x86_64";
```

Comment:
How come that this change _also_ appears here? Didn't we fix this long time ago?

Reply:
yes we did, but this branch is oooold

Reply:
There should be a GitHub button offering you to merge master into this branch. This should be such that see a clean diff between this branch and master. Unless @hellais has objections, I really suggest to use it.

Reply:
master already has more PRs merged in that, is still possible to do what you say without breaking everything?

Reply:
After the rebase this is still different. What is correct the one in this branch or the one in master?

Reply:
Why did you rebase?

Reply:
I didn't rebase, I updated from master. Wrong choice of terms

Reply:
Ah, okay. Well, let me double check what is going on here.

Reply:
`arm64` should be the good one and `armv64` the wrong one, so this should be good to merge

Reply:
@lorenzoPrimi I am starting to be quite confused at this point:

```
$ git branch
  feature/send_email
  fix/crashes
* master
$ git describe --tags
v2.0.5-22-g3cb279e7
$ git grep armv64
ooniprobe.xcodeproj/project.pbxproj:                            VALID_ARCHS = "armv64 armv7s i386 x86_64";
[0]dunkerque:probe-ios$ 
```

Am I using the wrong branch? I would have expected the issue to be fixed in master? Am I doing other doofus?

Reply:
I dont remember when it was fixed. Probably XCode override something?

Reply:
I don't know. But we should have a better process and not loose fixes. I guess the workflow with master and possibly many small PRs to master and code review is good to prevent that.